src/lib.rs

```diff
@@ -188,12 +188,12 @@
 //! }
 //! ```
 //!
-//! This example is also included in the [simulator](https://github.com/jamwaffles/embedded-graphics/tree/master/simulator/examples) crate and
+//! This example is also included in the [simulator](https://github.com/embedded-graphics/embedded-graphics/tree/master/simulator/examples) crate and
 //! can be run using `cargo run --example hello-world`. It produces this output:
 //!
-//! ![Embedded Graphics Simulator example screenshot](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/assets/hello-world-simulator.png)
+//! ![Embedded Graphics Simulator example screenshot](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/assets/hello-world-simulator.png)
 //!
-//! Additional examples can be found in the [simulator](https://github.com/jamwaffles/embedded-graphics/tree/master/simulator) crate.
+//! Additional examples can be found in the [simulator](https://github.com/embedded-graphics/embedded-graphics/tree/master/simulator) crate.
```

Comment:
This will also need to be changed after we move the examples (see comment above):
```suggestion
//! Additional examples can be found in the [simulator](https://github.com/embedded-graphics/simulator) crate.
```